### PR TITLE
Variable mapping

### PR DESCRIPTION
**Pull request by missjwo**
changes in this PR: 

1. Changing the hm footer link to turn brand red when focused. Also made the H linked too. 
2. Make page history a theme feature that can be turned on and off using `add_theme_support`. It is added by default in the hm-handbook theme.
3. updated pattern library version to 1.0.4 release tag

### CODE REVIEW

**Comment by missjwo** [comment]
Also now has the ability to add in custom site logos.

File: header.php [hunk 1]
```diff
@@ -30,11 +30,13 @@
 
 		<?php if ( is_front_page() ) : ?>
 			<div class="site-logo">
-
-				<a class="HMLogo HMLogo-White" href="<?php echo esc_url( home_url() ); ?>/">
-					Human Made Limited
-				</a>
-
+				<?php if ( function_exists( 'the_custom_logo' ) ) {
```
Comment (mattheu) [inline header.php]: Should this be `has_site_logo`?

Comment (missjwo) [inline header.php]: has_custom_logo .. yes but  docuemntation says that you shoudl check for old versionf o WordPress by checking if the fucntion exists too so i just added that bit. I dunno if you wanted to hve  aminimum WordPress version requirement.

**Comment by mattheu** [comment]
Thanks for this JennY!

**Pull request merged**